orc8r/cloud/docker/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
 #      DATABASE_SOURCE: "magma_dev:magma_dev@(maria)/magma_dev"
       SQL_DIALECT: psql
       SERVICE_REGISTRY_MODE: yaml
+      ORC8R_DOMAIN_NAME: LOCAL-DEV
```

Comment:
This should probably be `magma.test`

Reply:
Idk why I rebased this PR onto the domain name PR, but I just undid the rebase and fixed all these in the original domain name PR.